src/python/pants/backend/awslambda/python/awslambda_python_rules_test.py

```diff
@@ -20,6 +25,21 @@
 from pants.testutil.test_base import TestBase
 
 
+@pytest.mark.parametrize(
+    ["runtime", "expected_major", "expected_minor"],
+    (
+        # The available runtimes at the time of writing.
+        # See https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html.
+        ["python2.7", 2, 7],
+        ["python3.6", 3, 6],
+        ["python3.7", 3, 7],
+        ["python3.8", 3, 8],
+    ),
+)
+def test_get_interpreter_from_runtime(runtime, expected_major, expected_minor):
+    assert (expected_major, expected_minor) == get_interpreter_from_runtime(runtime, "")
+
```

Comment:
Bad test now. The idiom I've been doing is to create a `targets_test.py`. You would test something like:

```python
assert PythonAwsLambdaRuntime("python2.7", addres=addr).to_interpreter_version == (2, 7)
```

You'll also want:

```python
with pytest.raises(InvalidFieldException):
  PythonAwsLambdaRuntime("2.7", addres=addr)
```

etc, with a couple different values like `python3`, `python3.7.4`, and so on.